### ISSUE TITLE
3.0.3 fails to build on Arch

### ISSUE THREAD

**Issue by felixonmars** [ukui/ukui-media#64]
```
ukmedia_device_switch_widget.cpp: In static member function ‘static headset_ports* DeviceSwitchWidget::get_headset_ports(MateMixerStreamControl*, const pa_card_info*)’:
ukmedia_device_switch_widget.cpp:5553:40: error: invalid use of incomplete type ‘MateMixerStreamControlPrivate’ {aka ‘struct _MateMixerStreamControlPrivate’}
 5553 |     if (h->headphones && (control->priv->server_protocol_version >= 34)) {
      |                                        ^~
```

**Comment by handsome-feng** [comment]
Fixed by 4bbf7631b243840bdb4945acc3b3d4561cdba4d9 15 days ago. 
And I just make a tag v3.0.3.1.

**Comment by felixonmars** [comment]
Thanks. Confirmed that v3.0.3.1 works fine now.
